sem/database.py

```diff
@@ -206,6 +207,16 @@ def insert_results(self, results):
             'meta': {k: ['...'] for k in ['elapsed_time', 'id', 'exitcode']},
         }
 
+        # If logging is enabled, the results should have a log_component 
+        # dictionary inside the meta dictionary
+        if log_component is not None:
+            log_component_list = list(log_component.keys())
+            example_result['meta']['log_component'] = {
+                k: '...' for k in log_component_list
+            }
+        else:
+            example_result['meta']['log_component'] = None
+
         for result in results:
             # Verify result format is correct
             if not(DatabaseManager.have_same_structure(result, example_result)):
```

Comment:
> does this mean that all results should have the same log_component dictionary? this might be a problem for us, right?

It will not be a problem even in the current state, as I have modified the results accordingly in the `CampaignManager.run_and_save_results()`. 

> I think with my suggestion above we should be fine, as only the presence of the log_component key will be checked.

I am not sure if I understood your suggestion correctly, but I did reply to it above.